src/screens/General/Dashboard.js

```diff
@@ -273,226 +264,242 @@ class Dashboard extends Component {
-											style = {{ color: "#FECB00" }}
-										/>
-									</View>
-								</View>
-							</View>
-						</TouchableOpacity>
-					) }
-				</View>
+				{ content }
 			</View>
 		);
 	}
 
+	viewCommittee(item) {
+		this.props.loadCommittee(item);
+		Actions["CommitteePageD"]({ screen: "dashboard" });
+	}
+
 	renderButtonLinks() {
 		const {
-			containerStyle
+			socialMediaButton,
+			socialMediaContainer,
+			buttonRowContainer,
+			black
 		} = styles;
 
 		return (
-			<View style = {{ flex: 0.35, alignItems: "center" }}>
-				<View style = {{ flex: 0.1 }}></View>
-				<View style = {{ flexDirection: "row", justifyContent: "center", flex: 1, alignItems: "center" }}>
+			<View style = { socialMediaContainer }>
+				<View style = { buttonRowContainer }>
 					<TouchableOpacity
-						style = { containerStyle }
+						style = { socialMediaButton }
 						onPress = { () => Linking.openURL("https://shpeucf2018-2019.slack.com/") }
 					>
 						<FontAwesomeIcon
-							style = {{ color: "black" }}
+							style = { black }
 							name = "slack"
 							size = { dimension.height * 0.04 }
 						/>
 					</TouchableOpacity>
 					<TouchableOpacity
-						style = { containerStyle }
+						style = { socialMediaButton }
 						onPress = { () => Linking.openURL("https://www.facebook.com/shpeucfchapter/") }
 					>
 						<FontAwesomeIcon
-							style = {{ color: "black" }}
+							style = { black }
 							name = "facebook"
 							size = { dimension.height * 0.04 }
 						/>
 					</TouchableOpacity>
 					<TouchableOpacity
-						style = { containerStyle }
+						style = { socialMediaButton }
 						onPress = { () => Linking.openURL("https://www.shpeucf.com/") }
 					>
 						<FontAwesomeIcon
-							style = {{ color: "black" }}
+							style = { black }
 							name = "globe"
 							size = { dimension.height * 0.04 }
 						/>
 					</TouchableOpacity>
 					<TouchableOpacity
-						style = { containerStyle }
+						style = { socialMediaButton }
 						onPress = { () => Linking.openURL("https://www.instagram.com/shpeucf/?hl=en") }
 					>
 						<FontAwesomeIcon
-							style = {{ color: "black" }}
+							style = { black }
 							name = "instagram" size = { dimension.height * 0.04 }
 						/>
 					</TouchableOpacity>
 				</View>
-				<View style = {{ flex: 0.6 }}></View>
 				{ this.renderFooter() }
 			</View>
 		);
 	}
 
 	renderFooter() {
+		const {
+			footer,
+			footerText,
+			black,
+			textColor
+		} = styles;
+
 		return (
-			<View style = {{ flex: 0.3, justifyContent: "center", backgroundColor: "#FECB00", width: "100%" }}>
-				<View style = {{ flexDirection: "row", justifyContent: "center" }}>
-					<Text style = {{ color: "black" }}>SHPE </Text>
-					<Text style = {{ color: "white" }}>UCF</Text>
+			<View style = { footer }>
+				<View style = { footerText }>
+					<Text style = { black }>SHPE </Text>
+					<Text style = { textColor }>UCF</Text>
 				</View>
 			</View>
 		);
 	}
 
 	renderFlags() {
```

Comment:
I made a comment above about the flags being their own component but for now atleast in utils. Same for profile screen